### PR TITLE
add java images with zulu jdk for roll back options

### PR DESCRIPTION
**Pull request by kaibocai**
This pr adding java zulu images as java8.1 and java11.1 for roll back options, right now just copy and paste from default images. Later will create a new pr to update default java images to use new jdks. 

---

### PR information

- [X] The title of the PR is clear and informative.
- [X] There are a small number of commits, each of which has an informative message. This means that previously merged commits do not appear in the history of the PR. For information on cleaning up the commits in your pull request, [see this page](https://github.com/Azure/azure-powershell/blob/dev/documentation/development-docs/cleaning-up-commits.md).
- [ ] If applicable, the PR references the bug/issue that it fixes in the description.
- [ ] New Unit tests were added for the changes made and CI is passing.

### CODE REVIEW

File: host/4/bullseye/amd64/java/java11-zulu/java11-composite.template [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    DOTNET_USE_POLLING_FILE_WATCHER=true \
+    HOST_VERSION=${HOST_VERSION}
+
+# Fix from https://github.com/GoogleCloudPlatform/google-cloud-dotnet-powerpack/issues/22#issuecomment-729895157
+RUN apt-get update && \
+    apt-get install -y libc-dev
+
+COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
+COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
+COPY --from=jre [ "/usr/lib/jvm/zre-hl-tools-11-azure-amd64", "/usr/lib/jvm/zre-11-azure-amd64" ]
```
Comment (pragnagopa) [inline host/4/bullseye/amd64/java/java11-zulu/java11-composite.template:26]: @cooperlink - please verify if the copy cmds look good - to ensure layer reuse

Comment (CooperLink) [reply host/4/bullseye/amd64/java/java11-zulu/java11-composite.template:26]: Will sync with Kaibo offline. Need to move the func ext bundles.

Comment (kaibocai) [reply host/4/bullseye/amd64/java/java11-zulu/java11-composite.template:26]: @CooperLink updated the images as suggested, please take a look. thank you.

Comment (kaibocai) [reply host/4/bullseye/amd64/java/java11-zulu/java11-composite.template:26]: @pragnagopa @CooperLink , team please let me know if the pr looks good now. thank you. Really appreciate your help!

File: host/3.0/buster/amd64/java/java11-zulu/java11-composite.template [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+FROM mcr.microsoft.com/java/jre-headless:${JAVA_VERSION}-zulu-debian10-with-tools as jre
+FROM mcr.microsoft.com/dotnet/core/runtime-deps:3.1
+ARG HOST_VERSION
+
+COPY sshd_config /etc/ssh/
+COPY start.sh /azure-functions-host/
+COPY --from=runtime-image [ "/FuncExtensionBundles", "/FuncExtensionBundles" ]
+COPY --from=runtime-image [ "/azure-functions-host", "/azure-functions-host" ]
+COPY --from=runtime-image [ "/workers/java", "/azure-functions-host/workers/java" ]
```
Comment (CooperLink) [inline host/3.0/buster/amd64/java/java11-zulu/java11-composite.template:11]: This is looking good.

File: host/4/bullseye/amd64/java/java11-zulu/java11-composite.template (hunk 1 above)
Comment (CooperLink) [reply host/4/bullseye/amd64/java/java11-zulu/java11-composite.template:26]: Looks great to me. Thanks Kaibo

**Review by CooperLink — APPROVED** [review]
Looks great to me.  Thank you for your contribution Kaibo!

**Pull request merged**